data/json/recipes/tools/containers.json

```diff
@@ -507,6 +507,19 @@
     "using": [ [ "filament", 300 ], [ "cordage", 1 ] ],
     "components": [ [ [ "cotton_patchwork", 12 ] ] ]
   },
+  {
+    "result": "storage_line",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_CONTAINERS",
+    "skill_used": "tailor",
+    "difficulty": 1,
+    "time": "1 h",
+    "autolearn": true,
+    "using": [ [ "tailoring_nylon_patchwork", 2 ] ],
```

Comment:
Where did the carabiners on this go 🤔

Reply:
Fixed, recipe now needs some metal as well.